clang/lib/AST/ExprConstant.cpp

```diff
@@ -15550,6 +15586,17 @@ class VoidExprEvaluator
     case Builtin::BI__builtin_operator_delete:
       return HandleOperatorDeleteCall(Info, E);
 
+    case Builtin::BI__builtin_contract_assert: {
```

Comment:
FIXME: This also is unused and needs to be removed.

ContractStmts need to be handled differently.